packages/squiggle-lang/__tests__/SqValue/asJS_test.ts

```diff
@@ -0,0 +1,25 @@
+import { testRun } from "../helpers/helpers.js";
+
+describe("SqValue.asJS", () => {
+  test("SqDict -> Map", async () => {
```

Comment:
minor, but I don't get what this is meant to be testing. "Is instance of object" doesn't tell us much. Please improve the description or test.

Reply:
I didn't touch this code, just moved it (from `public/SqValue_test.ts`, which was becoming too long and unfocused).

The patch that caused the mismatch from the description is yours: https://github.com/quantified-uncertainty/squiggle/commit/5c1f6241244426c64fc175f98e56b342b35d2b9b

Reply:
Ah, makes sense